pkg/pci/pci_linux.go

```diff
@@ -112,6 +117,10 @@ func parseModaliasFile(fp string) *deviceModaliasInfo {
 }
 
 func parseModaliasData(data string) *deviceModaliasInfo {
+	// extra sanity check to avoid segfaults
+	if len(data) < modAliasExpectedLength {
```

Comment:
what if the data is greater than 54 chars? :)

Reply:
We should happily ignore the extra data, but this calls for both a comment and a test! Will add them!